helios/stats_views.py

```diff
@@ -40,13 +40,15 @@ def elections(request):
 
   page = int(request.GET.get('page', 1))
   limit = int(request.GET.get('limit', 25))
+  q = request.GET.get('q','')
```

Comment:
I see a XSS possibility here since `q` is not being sanitized before being rendered in HTML. Could you add some sanitization?